src/views/count.vue

```diff
@@ -216,7 +216,7 @@
         await picker.present();
       },
       updateProductInventoryCount() {
-        if (this.quantity) {
+        if(this.quantity) {
```

Comment:
We have not handled negative check in this case, and also we have added static text entry in locale but have added the toast anywhere